src/main/java/uk/gov/companieshouse/api/accounts/model/rest/notes/CreditorsWithinOneYear.java

```diff
@@ -0,0 +1,145 @@
+    @JsonProperty("bank_loans_and_overdrafts")
+    private Long bankLoansAndOverdrafts;
+
+    @Range(min=MIN_RANGE,max=MAX_RANGE, message = "value.outside.range")
+    @JsonProperty("finance_leases_and_hire_purchase_contracts")
+    private Long financeLeasesAndHirePurchaseContracts;
+
+    @Range(min=MIN_RANGE,max=MAX_RANGE, message = "value.outside.range")
+    @JsonProperty("other_creditors")
+    private Long otherCreditors;
+
+    @Range(min=MIN_RANGE,max=MAX_RANGE, message = "value.outside.range")
+    @JsonProperty("taxation_and_social_security")
+    private Long taxationAndSocialSecurity;
+
+    @Range(min=MIN_RANGE,max=MAX_RANGE, message = "value.outside.range")
+    @JsonProperty("trade_creditors")
+    private Long tradeCreditors;
+
+    @Range(min=MIN_RANGE,max=MAX_RANGE, message = "value.outside.range")
+    @JsonProperty("total")
+    private Long total;
+
+    @Size(max = MAX_FIELD_LENGTH, message = "max.length.exceeded")
+    @CharSetValid(CharSet.CHARACTER_SET_3)
+    @JsonProperty("details")
+    private String details;
+
+    public Long getAccrualsAndDeferredIncome() {
+        return accrualsAndDeferredIncome;
+    }
+
+    public void setAccrualsAndDeferredIncome(Long accrualsAndDeferredIncome) {
+        this.accrualsAndDeferredIncome = accrualsAndDeferredIncome;
+    }
+
+    public Long getBankLoansAndOverdrafts() {
+        return bankLoansAndOverdrafts;
+    }
+
+    public void setBankLoansAndOverdrafts(Long bankLoansAndOverdrafts) {
+        this.bankLoansAndOverdrafts = bankLoansAndOverdrafts;
+    }
+
+    public Long getFinanceLeasesAndHirePurchaseContracts() {
+        return financeLeasesAndHirePurchaseContracts;
+    }
+
+    public void setFinanceLeasesAndHirePurchaseContracts(Long financeLeasesAndHirePurchaseContracts) {
+        this.financeLeasesAndHirePurchaseContracts = financeLeasesAndHirePurchaseContracts;
+    }
+
+    public Long getOtherCreditors() {
+        return otherCreditors;
+    }
+
+    public void setOtherCreditors(Long otherCreditors) {
+        this.otherCreditors = otherCreditors;
+    }
+
+    public Long getTaxationAndSocialSecurity() {
+        return taxationAndSocialSecurity;
+    }
+
+    public void setTaxationAndSocialSecurity(Long taxationAndSocialSecurity) {
+        this.taxationAndSocialSecurity = taxationAndSocialSecurity;
+    }
+
+    public Long getTradeCreditors() {
+        return tradeCreditors;
+    }
+
+    public void setTradeCreditors(Long tradeCreditors) {
+        this.tradeCreditors = tradeCreditors;
+    }
+
+    public Long getTotal() {
+        return total;
+    }
+
+    public void setTotal(Long total) {
+        this.total = total;
+    }
+
+    public String getDetails() {
+        return details;
+    }
+
+    public void setDetails(String details) {
+        this.details = details;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Curly braces required by our coding standards even when they are syntactically not needed. Line 120 and line 121.

Reply:
done